l10n_br_account_product/models/account_invoice.py

```diff
@@ -139,10 +141,30 @@ def _compute_cfops(self):
                 lines |= line.cfop_id
         self.cfop_ids = (lines).sorted()
 
+    issuer = fields.Selection(
+        [('0', u'Emissão própria'), ('1', 'Terceiros')],
+        'Emitente',
+        default='0',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    # FIXME
```

Comment:
why FIXME? If there is something to be fixed, at least some comment would be welcome...